tutorials/mysql_setup.md

```diff
@@ -102,4 +102,8 @@ Fill the form, following this example:
 - Username - Username of the user we created earlier.
 - Password - Password of the user we created earlier.
 - Linked node - Node the database is hosted on. (Default value can be retained)
-Hit create and you are done!
\ No newline at end of file
+::: tip
+When your database server is on the same node as the game server use `172.18.0.1` and not `127.0.0.1`<br>
```

Comment:
When doing tips, always include the _why_ behind something, otherwise people just end up in Discord complaining that something isn't working because "the docs said do this" rather than the docs having pointers on what _might need to be done_.